Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.10-alpine
+LABEL maintainer="leon.kushnir15@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .

```

Comment:
Copying the entire context with `COPY . .` can lead to larger image sizes if unnecessary files are included. Ensure that your `.dockerignore` file is properly configured to exclude files that are not needed in the image.